test/branded_token/request_stake.js

```diff
@@ -0,0 +1,53 @@
+// Copyright 2018 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const BN = require('bn.js');
+const { AccountProvider } = require('../test_lib/utils.js');
+const { Event } = require('../test_lib/event_decoder.js');
+
+const utils = require('../test_lib/utils');
+const brandedTokenUtils = require('./utils');
+
+contract('TestBrandedToken::requestStake', async () => {
+    contract('Event', async (accounts) => {
+        const accountProvider = new AccountProvider(accounts);
+
+        it('Checks that StakeRequested event is emitted successfully.', async () => {
+            const {
+                testBrandedToken,
+            } = await brandedTokenUtils.setupTestBrandedToken(accountProvider);
+
+            const stakeAmount = 1;
+            const mintAmount = 5;
```

Comment:
Suggest making `mintAmount` 1.

Making `stakeAmount` and mintAmount different implies that they have to be different or should be different, neither of which is the case.